src/clndr.js

```diff
@@ -619,6 +619,18 @@
 
         if (eventsToday.length) {
             extraClasses += (" " + this.options.classes.event);
+            eventsToday.forEach( function( event){
+              if(event.extraClasses !== null){
+                if(Array.isArray(event.extraClasses)){
+                  event.extraClasses.forEach(function(extraClasses){
+                    extraClasses += (" "+extraClasses)
+                  })
+                }else{
+                  extraClasses += (" "+event.extraClasses)
+                }
+
+              }
+            } )
```

Comment:
This looks pretty verbose for what I think is just a space delineated concatenation of the classes.

What if it was just:

```
extraClasses += (" " + this.options.classes.event)
    + (_.isArray(event.extraClasses) ? " " + event.extraClasses.join(" ") : "");
```
?